strings.txt

```diff
@@ -16217,48 +16217,644 @@
+    fi = Tämä näyttää ehdottamiesi muutosten lukumäärän tällä hetkellä vain sinun laitteessasi saatavissa olevassa kartassa.
+    cs = Zde vidíte počet navržených změn, které jsou v tuto chvíli dostupné pouze na Vašem zařízení.
+    it = Qui è visualizzato il numero di modifiche suggerite sulla mappa che sono attualmente solo accessibili sul tuo dispositivo.
+    nb = Dette viser antallet foreslåtte endringer på kartet som i øyeblikket bare er tilgjengelige på enheten din.
+    zh-Hans = 这显示了当前仅在您设备上能够访问的对地图建议修改的数量。
+    th = หมายเลขที่แสดงอยู่ คือ หมายเลขที่คุณได้ทำการแนะนำการเปลี่ยนแปลงบนแผนที่ที่สามารถเข้าถึงได้โดยอุปกรณ์ของคุณท่านั้น
+    ja = これは、現在お使いのデバイスでのみアクセス可能な、マップ上に提案された変更の数を示しています。
+    ro = Acesta arată numărul de modificări sugerate pe hartă care sunt momentan accesibile doar pe dispozitivul dvs.
+    ar = يوضح هذا عدد التغييرات المقترحة على الخريطة والتي يمكن الوصول إليها حاليًا فقط من على جهازك.
+    sk = Tu sa zobrazuje počet navrhovaných zmien na mape momentálne dostupných len vo vašom zariadení.
+    es = Esto muestra el número de sus cambios sugeridos en el mapa que actualmente solo son accesibles desde su dispositivo.
+    pl = Jest to liczba sugerowanych przez ciebie zmian na mapie, które aktualnie dostępne są tylko na twoim urządzeniu.
+    nl = Dit toont het aantal voorgestelde wijzigingen voor de kaart die momenteel alleen toegankelijk zijn op uw apparaat.
+
+  [editor_international_names_subtitle]
+    comment = Подзаголовок в редакторе
+    en = Name in other languages
+    ru = Название на других языках
+    fr = Nom dans d'autres langues
+    da = Navn på andre sprog
+    id = Nama dalam bahasa lain
+    ko = 다른 언어로된 이름
+    sv = Namn på andra språk
+    tr = Diğer dillerdeki adı
+    uk = Назва на інших мовах
+    vi = Tên bằng các ngôn ngữ khác
+    hu = Név más nyelveken
+    de = Name in anderen Sprachen
+    fi = Nimi toisella kielellä
+    cs = Název v jiných jazycích
+    it = Nome in altre lingue
+    nb = Navn på andre språk
+    zh-Hans = 其他语言的名称
+    th = ชื่อในภาษาอื่น
+    ja = 他の言語での名前
+    ro = Nume în alte limbi
+    ar = الاسم بلغات أخرى
+    sk = Názov v iných jazykoch
+    es = Nombre en otros idiomas
+    pl = Nazwa w innych językach
+    nl = Naam in andere talen
+
+  [editor_send_to_osm]
+    comment = Button (Action Sheet)
+    en = Send to OSM
+    ru = Отправить в OSM
+    fr = Envoyer à OSM
+    da = Send til OSM
+    id = Kirim ke OSM
+    ko = OSM에 전송
+    sv = Skicka till OSM
+    tr = OSM’ye gönder
+    uk = Відправити до OSM
+    vi = Gửi đến OSM
+    hu = Küldés az OSM-nek
+    de = An OSM senden
+    fi = Lähetä OSM:ään
+    cs = Odeslat do OSM
+    it = Invia a OSM
+    nb = Send til OSM
+    zh-Hans = 发送至 OSM
+    th = ส่งไปที่ OSM
+    ja = OSMに送信
+    ro = Trimitere la OSM
+    ar = إرسال الى OSM
+    sk = Odoslať do OSM
+    es = Enviar a OSM
+    pl = Wyślij do OSM
+    nl = Naar OSM sturen
+
+  [editor_remove]
+    en = Remove
```

Comment:
В каких тегах?

> On Apr 26, 2016, at 5:49 PM, Igor Tomko <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61099418>:
> 
> > +    fi = Lähetä OSM:ään
> > +    cs = Odeslat do OSM
> > +    it = Invia a OSM
> > +    nb = Send til OSM
> > +    zh-Hans = 发送至 OSM
> > +    th = ส่งไปที่ OSM
> > +    ja = OSMに送信
> > +    ro = Trimitere la OSM
> > +    ar = إرسال الى OSM
> > +    sk = Odoslať do OSM
> > +    es = Enviar a OSM
> > +    pl = Wyślij do OSM
> > +    nl = Naar OSM sturen
> > +
> > +  [editor_remove]
> > +    en = Remove
> В тегах есть место и что это
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295#r61099418>